controllers/auth.js

```diff
@@ -1,15 +1,19 @@
 import User from "../models/User.js";
 import bcrypt from "bcryptjs";
 import { createToken, verifyToken } from "../utils/jwtHelper.js";
+const Joi = require('joi')
 
 export const loginHandler = async (req, res) => {
   try {
     const { email, password } = req.body;
 
-    if (!email || !password) {
-      return res
-        .status(400)
-        .json({ message: "Email and password are required" });
+    const schema = Joi.object().keys({
```

Comment:
Move this schema inside validators directory here: https://github.com/MLSAKIIT/devblogs-backend/tree/main/validators

Example: https://github.com/MLSAKIIT/devblogs-backend/blob/main/validators/register.js

Reply:
Thanks for your suggestion